src/main/java/doob/model/powerup/PlayerSpeedPowerUp.java

```diff
@@ -25,7 +24,7 @@ public void onActivate(Level level, Player player) {
     }
 
     /**
-     * Does nothing.
+     * Sets player speed back to initial value.
      * @param level the level the power-up is in.
```

Comment:
This javadoc is called something like Return to normal state in the freeze powerup. Maybe caal this one the same?